### PR TITLE
Requested Changes & New Sections Implemented

### PR DESCRIPTION
**Pull request by kaifur-rahman**
**New Sections Added:**
----------------------------------------------
1) About Section 
2) Events Section
3) Companies Section
4) Login Page

**Changes Addressed from the last PR review.**
-------------------------------------------------------
1) The link tag referencing the manifest.json has been removed
2) The robots.txt file has been restored
3) reportWebVitals.js has been restored
4) Minimal components ending with a parent have been restructured in the project directory and are moved to the folder called views which denotes the starting point of each component.

### CODE REVIEW

File: frontend/public/c1.png
Comment (ankitamk14) [inline frontend/public/c1.png:1]: Please remove this file from git.

File: frontend/public/c2.png
Comment (ankitamk14) [inline frontend/public/c2.png:1]: Please remove this file from git.

File: frontend/public/c3.jpg
Comment (ankitamk14) [inline frontend/public/c3.jpg:1]: Please remove this file from git.

File: frontend/public/c4.png
Comment (ankitamk14) [inline frontend/public/c4.png:1]: Please remove this file from git.

File: frontend/public/c5.png
Comment (ankitamk14) [inline frontend/public/c5.png:1]: Please remove this file from git.

File: frontend/public/c6.jpg
Comment (ankitamk14) [inline frontend/public/c6.jpg:1]: Please remove this file from git.

File: frontend/public/company-logo.png
Comment (ankitamk14) [inline frontend/public/company-logo.png:1]: Please remove this file from git.

File: frontend/public/favicon.ico
Comment (ankitamk14) [inline frontend/public/favicon.ico:1]: Please remove this file from git.

File: frontend/public/logo192.png
Comment (ankitamk14) [inline frontend/public/logo192.png:1]: Please remove this file from git.

**Comment by kaifur-rahman** [comment]
All the suggested changes have been made.

**Review by ankitamk14 — APPROVED** [review]

**Pull request merged**